CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Latest
+
+### Added
+
+* Add support for `Client(allow_redirect=<bool>)`. (Pull #1790)
```

Comment:
Typo: Is it allow_redirect**S**?

Reply:
It is indeed! That one got added prior to https://github.com/encode/httpx/pull/1808. 😬